mariamihaelafainarea/assignment2/LowestAncestorTest

```diff
@@ -0,0 +1,77 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+/**
+ *
+ * @author maria
+ */
+public class LowestAncestorTest {
+
+    public LowestAncestor<Integer> getTree() {
```

Comment:
You are using getTree() in all your test cases, this is a good example of when you can use @Before annotated method. It will prepare all your tests and you won't have to duplicate the code.

Reply:
done.